source/AI.cpp

```diff
@@ -1433,7 +1433,7 @@ void AI::MoveIndependent(Ship &ship, Command &command) const
 		int jumps = ship.JumpsRemaining(false);
```

Comment:
We're now returning a bool to an int variable that we only ever use as a bool. (Lines 1436 and 1442 are just checking that jumps is non-zero.)